app/controllers/foods_controller.rb

```diff
@@ -0,0 +1,69 @@
+class FoodsController < ApplicationController
+  before_action :set_food, only: %i[show edit update destroy]
+
+  # GET /foods or /foods.json
+  def index
+    @foods = Food.all
+  end
+
+  # GET /foods/1 or /foods/1.json
+  def show; end
+
+  # GET /foods/new
+  def new
+    @food = Food.new
+  end
+
+  # GET /foods/1/edit
+  def edit; end
```

Comment:
Good work using scaffold. The requirements point out that we shouldn't implement the update action so please let's remove these actions "edit" and "update"

Reply:
Done